src/main/resources/application-aat-STAGING-slot.yaml

```diff
@@ -0,0 +1,3 @@
+---
+job.scheduler:
+  autoStart: false
```

Comment:
Shouldn't this be true? As the reminder tests run against this slot so don't think they will pass if set to false?